artifacts/pom.xml

```diff
@@ -212,13 +212,17 @@
 		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
 
 		<!-- certify mock plugin -->
-		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
 		<certify-mock-plugin.version>0.3.0-SNAPSHOT</certify-mock-plugin.version>
 		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
 
 		<!-- certify mock ida dataprovider plugin -->
 		<certify-mock-ida-dataprovider-plugin.version>0.3.0-SNAPSHOT</certify-mock-ida-dataprovider-plugin.version>
-		<certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
+        <certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
```

Comment:
All mocks to be part of the mock-certify-plugin, why do we have a different entry called certify-mock-ida-dataprovider-plugin.jar ?

Reply:
@vharsh Is suppose to create a bug for this and attach to this comment, marking it resolved for now.

Reply:
https://mosip.atlassian.net/browse/INJICERT-643